cli/tests/pcluster/config/test_validators.py

```diff
@@ -484,16 +484,20 @@ def test_raid_validators(mocker, section_dict, expected_message):
 @pytest.mark.parametrize(
     "section_dict, expected_message",
     [
-        ({"imported_file_chunk_size": 1024, "import_path": "test"}, None),
+        ({"imported_file_chunk_size": 1024, "import_path": "test", "storage_capacity": 1200}, None),
         (
-            {"imported_file_chunk_size": 1024},
+            {"imported_file_chunk_size": 1024, "storage_capacity": 1200},
             "When specifying 'imported_file_chunk_size', the 'import_path' option must be specified",
         ),
-        ({"export_path": "test", "import_path": "test"}, None),
-        ({"export_path": "test"}, "When specifying 'export_path', the 'import_path' option must be specified"),
-        ({"shared_dir": "NONE"}, "NONE cannot be used as a shared directory"),
-        ({"shared_dir": "/NONE"}, "/NONE cannot be used as a shared directory"),
-        ({"shared_dir": "/fsx"}, None),
+        ({"export_path": "test", "import_path": "test", "storage_capacity": 1200}, None),
+        (
+            {"export_path": "test", "storage_capacity": 1200},
+            "When specifying 'export_path', the 'import_path' option must be specified",
+        ),
+        ({"shared_dir": "NONE", "storage_capacity": 1200}, "NONE cannot be used as a shared directory"),
+        ({"shared_dir": "/NONE", "storage_capacity": 1200}, "/NONE cannot be used as a shared directory"),
+        ({"shared_dir": "/fsx"}, "the 'storage_capacity' option must be specified"),
```

Comment:
Probably need to change this or add more tests according to comment above.

Reply:
That's done here https://github.com/aws/aws-parallelcluster/blob/develop/cli/tests/pcluster/config/test_validators.py#L714